web/ui/react-app/src/pages/LabelsTable.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { Button, Badge, Table } from 'reactstrap';
+
+interface LabelProps {
+  value: Record<string, Record<string, string>>[];
+  name: string;
+}
+
+const printLabels = (labels: Record<string, string>) => {
+  return Object.entries(labels).map(([key, value]) => {
+    return (
+      <div key={key}>
```

Comment:
I think you can omit the extra `div` wrapper here and put the key directly onto the `Badge`.

Reply:
Ah, you have the div for line breaks. Fair enough.

Reply:
yes :)